src/neo/SmartContract/ApplicationEngine.OpCodePrices.cs

```diff
@@ -186,6 +186,8 @@ partial class ApplicationEngine
             [OpCode.MIN] = 1 << 3,
             [OpCode.MAX] = 1 << 3,
             [OpCode.WITHIN] = 1 << 3,
+            [OpCode.PACKMAP] = 1 << 11,
```

Comment:
Maybe `PACKMAP` consume more resources

Reply:
Any one can test it?

Reply:
@neo-project/ngd-shanghai ?

Reply:
According to test it seems PACKMAP consume the same time as PACK